index.html

```diff
@@ -0,0 +1,187 @@
+                    <li class="element2">Back End Dev</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="bottom-list">
+                    <li><button class="language-buttons" type="button">html</button></li>
+                    <li><button class="language-buttons" type="button">css</button></li>
+                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                </ul>
+                <button class="p-button" type="button">See project</button>
+            </div>
+            <div class="flex-card">
+                <img class="snapshot" src="Snapshoot2Portfolio.svg" alt="portfolio 1">
+                <h1>Multi-Post Stories</h1>
+                <ul>
+                    <li class="element1">CANOPY</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">Back End Dev</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="bottom-list">
+                    <li><button class="language-buttons" type="button">html</button></li>
+                    <li><button class="language-buttons" type="button">css</button></li>
+                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                </ul>
+                <button class="p-button" type="button">See project</button>
+            </div>
+            <div class="flex-card">
+                <img class="snapshot" src="Snapshoot3Portfolio.svg" alt="portfolio 1">
+                <h1>Tonic</h1>
+                <ul>
+                    <li class="element1">CANOPY</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">Back End Dev</li>
+                    <li><img src="Counter.svg" alt="dot"></li>
+                    <li class="element2">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; 
+                    no accounts or sign-ups required.</p>
+                <ul class="bottom-list">
+                    <li><button class="language-buttons" type="button">html</button></li>
+                    <li><button class="language-buttons" type="button">css</button></li>
+                    <li><button class="language-buttons" type="button">javaScript</button></li>
+                </ul>
+                <button class="p-button" type="button">See project</button>
+            </div>
+            <div class="flex-card">
```

Comment:
- [Optional] Please consider using an indentation of 2 spaces for readability in all your documents. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).